engine/src/test/scala/cromwell/RetryableCallsSpec.scala

```diff
@@ -27,25 +25,27 @@ class RetryableCallsSpec extends CromwellTestkitSpec with Mockito {
     }
   }
 
+  val customizedLocalBackendOptions =
+    """
+      |{
+      |  "backend": "customizedLocalBackend"
+      |}"""".stripMargin
```

Comment:
Yeah I'm curious about that too... some parsers are actually okay with extraneous characters... but I feel like most are not.  Very odd.